astrodb_utils/__init__.py

```diff
@@ -11,13 +11,14 @@
 )
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
 
 LOGFORMAT = logging.Formatter(
-    "%(asctime)s %(levelname)s: %(message)s", datefmt="%m/%d/%Y %I:%M:%S%p"
-)
+    "%(name)-15s: %(levelname)-8s %(message)s")

```

Comment:
For reference, in some other code bases I use:
```python
LOGFORMAT = logging.Formatter("%(asctime)-15s - %(levelname)s - %(name)s - %(message)s")
LOGFORMAT.converter = time.gmtime  # ensure time in UTC
```
But If you explicitly removed asctime you probably don't want/need a timestamp here.